.ci/static-checks.sh

```diff
@@ -838,6 +839,58 @@ static_check_vendor()
 	dep ensure -no-vendor -dry-run
 }
 
+static_check_xml()
+{
+	local all_xml
+	local files
+
+	all_xml=$(find . -name "*.xml" | grep -v "vendor/" | sort || true)
```

Comment:
nitpick: don't know if `^vendor/` would be more accurate or not?

Reply:
I'll tal at this. We have a number of vendor-filtering points in the code, all subtly different... ;)